applications/nrf5340_audio/tools/buildprog/buildprog.py

```diff
@@ -42,9 +42,7 @@
         os.getenv("AUDIO_KIT_SERIAL_NUMBERS_JSON"))
 TARGET_BOARD_NRF5340_AUDIO_DK_APP_NAME = "nrf5340_audio_dk/nrf5340/cpuapp"
 
-TARGET_CORE_APP_FOLDER = NRF5340_AUDIO_FOLDER
-TARGET_DEV_HEADSET_FOLDER = NRF5340_AUDIO_FOLDER / "build/dev_headset"
-TARGET_DEV_GATEWAY_FOLDER = NRF5340_AUDIO_FOLDER / "build/dev_gateway"
+TARGET_AUDIO_FOLDER = NRF5340_AUDIO_FOLDER
```

Comment:
General: We need to add a note to the changelog about this buildprog change.
Suggest that we also add a deprecation note there. Something along the lines of:
"The buildprog.py is a convenience script for building and flashing multiple kits and cores with various audio application configurations. With time, we will progress towards using only standard tools for building and flashing kits. Buildprog will be deprecated in a future release."

Reply:
> The :file:`buildprog.py` is an app-specific script for building and flashing multiple kits and cores with various audio application configurations. The script will be deprecated in a future release. The audio applications will gradually shift only to using standard tools for building and programming development kits.

Reply:
@gWacey looks good! Though a note in the changelog is still required: `ncs/nrf/doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst`
regarding the change of build folder and deprecation.

Reply:
Should the new build directory location live in release or migration guide?